app/gui2/src/components/visualizations/JSONVisualization.vue

```diff
@@ -53,7 +57,11 @@ function createProjection(path: (string | number)[][]) {
 <template>
   <VisualizationContainer :belowToolbar="true">
     <div class="JSONVisualization">
```

Comment:
I'd suggest a CSS approach:
- Here at the top level, add `:class={ viewonly: !isClickThroughEnabled }`
- Define the class as `.viewonly { pointer-events: none; }`

That way we ensure we're disabling everything (like the hover titles), while keeping the rest of the logic simple.

(We are planning to add some view-only interactivity some day that shouldn't be disabled (like subtree expand/collapse)--when we do that, we can use another CSS class to mark the new `alwaysInteractive` elements.)